GlimpseImputationPipeline/Glimpse2SplitReference.wdl

```diff
@@ -0,0 +1,166 @@
+version 1.0
+
+workflow Glimpse2SplitReference {
+    input {
+        # There are two variables required to set define the contigs. The first one, contig_regions, defines the coordinates for the separate chunks,
+        # usually this would be an array of ["chr1", "chr2", ..., "chr22", "chrX:1-2781479", "chrX:2781480-155701382", "chrX:155701383-156030895"].
+        # Notice the split of chrX into PAR1, non-PAR, PAR2. When constructing the reference panel chunks, however, we need to determine the appropriate
+        # reference panel file names. We could do that by parsing the substring before the colon (which WDL doesn't support), or we can just pass
+        # another array contig_names_in_reference_panel = ["chr1", "chr2", ..., "chr22", "chrX", "chrX", "chrX"]. Note that contig_regions and
+        # contig_names_in_reference_panel must have the same length.
+        Array[String] contig_regions
+        Array[String] contig_names_in_reference_panel
+        Array[String] contig_names_in_genetic_maps
+
+        # Example path for chr1: gs://bucket/to/panel/reference_panel_chr1_merged.bcf(.csi)
+        # reference_panel_prefix = "gs://bucket/to/panel/reference_panel_"
+        # reference_panel_suffix = "_merged.bcf"
+        # reference_panel_index_suffix = ".csi"
+        String reference_panel_prefix
+        String reference_panel_suffix
+        String reference_panel_index_suffix
+
+        # Same format as reference_panel_pre-/suffix
+        String genetic_map_path_prefix
+        String genetic_map_path_suffix
+
+        Int? seed
+        Array[Float] min_window_cms
```

Comment:
do we really need the capability to use a different value for each contig?  I think this was to get around a bug at some point?

Reply:
This was to better control the sizes when using uniform_number_variants, because they would be uniform within a chromosome but not across chromosomes. I changed it back to one global setting.